provisioning/provisioning-service-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/service/auth/ProvisioningConnectionString.java

```diff
@@ -3,32 +3,53 @@
 
 package com.microsoft.azure.sdk.iot.provisioning.service.auth;
 
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+
 /**
  * Expose all connections string properties and methods
  * for user, device and connection string serialization.
  */
 public class ProvisioningConnectionString extends ProvisioningConnectionStringBuilder
 {
-    protected static final String VALUE_PAIR_DELIMITER = ";";
-    protected static final String VALUE_PAIR_SEPARATOR = "=";
-    protected static final String HOST_NAME_SEPARATOR = ".";
+    static final String VALUE_PAIR_DELIMITER = ";";
+    static final String VALUE_PAIR_SEPARATOR = "=";
+    static final String HOST_NAME_SEPARATOR = ".";
 
-    protected static final String HOST_NAME_PROPERTY_NAME = "HostName";
-    protected static final String SHARED_ACCESS_KEY_NAME_PROPERTY_NAME = "SharedAccessKeyName";
-    protected static final String SHARED_ACCESS_KEY_PROPERTY_NAME = "SharedAccessKey";
-    protected static final String SHARED_ACCESS_SIGNATURE_PROPERTY_NAME = "SharedAccessSignature";
+    static final String HOST_NAME_PROPERTY_NAME = "HostName";
+    static final String SHARED_ACCESS_KEY_NAME_PROPERTY_NAME = "SharedAccessKeyName";
+    static final String SHARED_ACCESS_KEY_PROPERTY_NAME = "SharedAccessKey";
+    static final String SHARED_ACCESS_SIGNATURE_PROPERTY_NAME = "SharedAccessSignature";
 
     private static final String USER_SEPARATOR = "@";
     private static final String USER_SAS = "SAS.";
     private static final String USER_ROOT = "root.";
 
     // Included in the device connection string
-    protected String hostName;
-    protected String deviceProvisioningServiceName;
-    protected AuthenticationMethod authenticationMethod;
-    protected String sharedAccessKeyName;
-    protected String sharedAccessKey;
-    protected String sharedAccessSignature;
+    @Getter
+    @Setter(AccessLevel.MODULE)
```

Comment:
q - is there a reason you chose module instead of package?
looks like they mean the same thing: https://github.com/projectlombok/lombok/blob/e050dafc9016519e79184e383eab9ffbd579bebd/src/core/lombok/javac/handlers/JavacHandlerUtil.java#L845-L847

Reply:
No reason in particular, but I can unify them to "PACKAGE" if that would make more sense

Reply:
I think it'll be good to use one throughout, esp since they fall into the same case. 
+1 to unifying them to "package", since it's closer to java's package package-private access level

(On a related note, I'd have thought Lombok AccessLevel.None would have been java-package-private (no modifier). Interesting that it is "private" instead).

Reply:
Sure thing, I'll mark all of these as PACKAGE for consistency